### PR TITLE
feat: allow passing command line arguments that take no value.

### PR DESCRIPTION
**Pull request by mprobst**

### CODE REVIEW

**Comment by mprobst** [comment]
@tim-smart passing `null` for void arguments is maybe not super elegant, but it's pragmatic; everything else looks uglier or would break backwards compat of the interface. Let me know if you have a better idea.

**Comment by tim-smart** [comment]
A boolean would make more sense I think. Not sure how you would handle the `false` case however?

**Comment by mprobst** [comment]
@tim-smart yeah, that's why I didn't go that way originally.

However, on second thought, it probably makes sense to just leave off flags that are passed with `false`. That makes sense if you e.g. have a tool that programatically generates these options, it's easier to handle the boolean absent/exists case then.

So, changed. PTAL.

File: src/index.coffee [hunk 1]
```diff
@@ -25,7 +25,7 @@ spawn = require('child_process').spawn
 path  = require 'path'
 
 JAVA_PATH = exports.JAVA_PATH = 'java'
-JAR_PATH  = exports.JAR_PATH  = path.join __dirname, 'node_modules/google-closure-compiler/compiler.jar'
+JAR_PATH  = exports.JAR_PATH  = path.join __dirname, '../node_modules/google-closure-compiler/compiler.jar'
```
Comment (mprobst) [inline src/index.coffee]: NB: this was apparently a leftover from previously, where an author changed the path in `index.js` (the generated file), but not here.

**Comment by tim-smart** [comment]
Do you have npm access? I'll add you as a publisher

**Comment by mprobst** [comment]
Thanks! I'm https://www.npmjs.com/~mprobst.

Tim <notifications@github.com> schrieb am Mi., 28. Okt. 2015 um 20:51 Uhr:

**Comment by tim-smart** [comment]
Cool, feel free to push a new version

**Comment by mprobst** [comment]
Done.

Tim <notifications@github.com> schrieb am Do., 29. Okt. 2015 um 08:41 Uhr:

**Pull request merged**